File: Week 01/id_591/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let i=0,j=0;
    for(i=0; i<nums.length-1; i++){
        if(nums[i] != nums[i+1]){
            j++;
            nums[j] = nums[i+1];
        }
    }
    nums = nums.splice(j+1);
    return j+1;
};

File: Week 01/id_011/remove-duplicates-from-sorted-array.go
package algorithm00401

func removeDuplicates(nums []int) int {
	i := 0
	for j := 0; j < len(nums); j++ {
		if nums[i] != nums[j] {
			i++
			nums[i] = nums[j]
		}
	}
	return i + 1
}
